soccer/src/soccer/strategy/agent/position/free_kicker.cpp

```diff
@@ -0,0 +1,41 @@
+#include "free_kicker.hpp"
+
+namespace strategy {
+
+FreeKicker::FreeKicker(int r_id) : Position(r_id, "FreeKicker") {}
+
+FreeKicker::FreeKicker(const Position& other) : Position{other} { position_name_ = "FreeKicker"; }
+
+std::optional<RobotIntent> FreeKicker::derived_get_task(RobotIntent intent) {
+    // Penalty Kicker kicks the ball into the goal
+
+    SPDLOG_INFO("Free Kicker {} is running", this->robot_id_);
```

Comment:
I'm not sure if I am a good judge of whether or not a log message is needed, but I'm wondering if there may be enough log messages throughout the code to clutter the logs a bit. Could some of these be commented out?